test/spec/search/createlocationsearchbloodhound.spec.js

```diff
@@ -1,22 +1,28 @@
 /* global geoAdminLocationSearch */
+
 goog.require('ngeo.search.createLocationSearchBloodhound');
 goog.require('ngeo.test.data.geoAdminLocationSearch');
 goog.require('ngeo.search.module');
 goog.require('ngeo');
+goog.require('ol.proj');
+
 
 describe('ngeo.search.createLocationSearchBloodhound', () => {
 
-  let ngeoCreateLocationSearchBloodhound;
+  let ngeoCreateLocationSearchBloodhound_;
   ngeo.module.requires.push(ngeo.search.module.name);
+  angular.module('myTest', [ngeo.module.name]);
+
+  beforeEach(angular.mock.module('myTest'));
 
   beforeEach(() => {
-    inject(($injector) => {
-      ngeoCreateLocationSearchBloodhound = $injector.get('ngeoCreateLocationSearchBloodhound');
+    inject((ngeoCreateLocationSearchBloodhound) => {
```

Comment:
See https://docs.angularjs.org/guide/unit-testing.
The Angular idiom is to wrap the name with underscores. Then do not use underscores for the variable.